pkg/kubectl/cmd/create_service.go

```diff
@@ -80,6 +80,7 @@ func NewCmdCreateServiceClusterIP(f cmdutil.Factory, cmdOut io.Writer) *cobra.Co
 	cmdutil.AddGeneratorFlags(cmd, cmdutil.ServiceClusterIPGeneratorV1Name)
 	addPortFlags(cmd)
 	cmd.Flags().String("clusterip", "", i18n.T("Assign your own ClusterIP or set to 'None' for a 'headless' service (no loadbalancing)."))
+	cmd.Flags().StringP("selector", "l", "", "Label Selector to define set of pods targeted by service, supports only '='.")
```

Comment:
@eparis : Yes it is a little confusing if I read it again.
What I want to say is : For this particular command selector only supports "equality" operation in argument
--selector='a=b'  or -l='a=b' --selector 'a=b'  or -l 'a=b' 

Contrary to this the parent command supports arguments with  '=', '==', and '!='.
I hope #45461 is not causing confusion